app/workers/asset_manager_create_asset_worker.rb

```diff
@@ -0,0 +1,50 @@
+class AssetManagerCreateAssetWorker < WorkerBase
+  include AssetManager::ServiceHelper
+
+  sidekiq_options queue: "asset_manager"
+
+  def perform(temporary_location, model_id, asset_variant, draft = false, attachable_model_class = nil, attachable_model_id = nil, auth_bypass_ids = [])
+    return unless File.exist?(temporary_location)
+
+    file = File.open(temporary_location)
+
+    asset_options = { file:, auth_bypass_ids: }
+    asset_options[:draft] = true if draft
```

Comment:
do we need to have this in a separate line?

Reply:
moved asset option to the asset_options = { file:, auth_bypass_ids: } to look like asset_options = { file:, auth_bypass_ids:, draft: } and removed line 12